### PR TITLE
Force SDL not to disable screensaver

### PR DESCRIPTION
**Pull request by piernov**
Dunno if it works but sending this out as a PR anyway. Seems like the issue #93 completely disapperead…

Anyway, test builds here for anyone who wants to try out: https://github.com/piernov/OpenBoardView/releases/tag/N15

### CODE REVIEW

**Comment by inflex** [comment]
Thanks @piernov .  Will check it later when I'm done with today's work.

File: src/openboardview/main_opengl.cpp [hunk 1]
```diff
@@ -360,9 +360,11 @@ int main(int argc, char **argv) {
 	cleanupAndExit(1);
 #endif
 
+	// SDL disables screen saver by default which doesn't make sense for us.
+	SDL_EnableScreenSaver();
+
 	ImGuiIO &io    = ImGui::GetIO();
 	io.IniFilename = NULL;
-	//	io.Fonts->AddFontDefault();
```
Comment (MattSturgeon) [inline src/openboardview/main_opengl.cpp]: Being nit-picky (as always!) but wouldn't this be better off in a separate cleanup commit? Probably ok to keep it in the PR though.

**Comment by piernov** [comment]
Any feedback on this?

**Comment by inflex** [comment]
Haven't had a chance to test yet @piernov , been too busy :(

**Comment by srakitnican** [comment]
I've tested binary from RPM package. It does not inhibit screen blanking on Fedora 25.

**Comment by piernov** [comment]
The issue was showing up mainly on macOS, Windows was affected when the window was focused. Not sure if it was doing anything on Linux.

**Comment by srakitnican** [comment]
Well, R7.3 that I've packaged for Fedora inhibits screen blanking.

**Comment by MattSturgeon** [comment]
Sounds like the fix works then, given the same setup can reproduce the issue without this PR but can't with it.

**Pull request merged**